lib/libutils/isoc/bget_malloc.c

```diff
@@ -782,38 +791,21 @@ static void free_helper(void *ptr, bool wipe)
 
 void *calloc(size_t nmemb, size_t size)
 {
-	void *p;
-	uint32_t exceptions = malloc_lock(&malloc_ctx);
-
-	p = raw_calloc(0, 0, nmemb, size, &malloc_ctx);
-	malloc_unlock(&malloc_ctx, exceptions);
-	return p;
+	return mem_alloc(MAF_ZERO_INIT, NULL, 1, nmemb, size);
 }
 
-static void *realloc_unlocked(struct malloc_ctx *ctx, void *ptr,
-			      size_t size)
-{
-	return raw_realloc(ptr, 0, 0, size, ctx);
-}
+/* For use in raw_malloc_add_pool() below */
+#define realloc_unlocked(ctx, ptr, size) \
+	raw_mem_alloc(MAF_NULL, ptr, 0, 0, 1, 1, size, ctx)
```

Comment:
Keeping a static function is preferable IMO

Reply:
This is an intermediate change that in the end turns into:
``` C
#define realloc_unlocked(ctx, ptr, size)                                      \
	mem_alloc_unlocked(MAF_NULL, (ptr), 1, 1, (size), __FILE__, __LINE__, \
			   (ctx))
```
This one should preferably be a macro for usable file and line info. I can rework the intermediate change if it's worth it.